src/submission/submission-api.ts

```diff
@@ -51,12 +54,13 @@ class SubmissionController {
       creator: creator,
       records: records,
       batchName: file.originalname,
+      fieldNames: Object.keys(records[0]), // every record in a tsv should have same fieldNames
```

Comment:
what happens if the file only have a headers row ?

Reply:
records is an array of map right? so `Object.keys(records[x])` should be the headers for the record map, if the values are missing it'll be a tsv parse error

Reply:
ok fair